community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid/README.md

```diff
@@ -0,0 +1,204 @@
+> installed on the system deploying the module:
+>
+> * [terraform]
+> * [addict]
+> * [google-cloud-pubsub]
+> * A full list of recommended python packages is available in a
+>   [requirements.txt] file in the [slurm-gcp] repo.
+
+[terraform]: https://learn.hashicorp.com/tutorials/terraform/install-cli
+[addict]: https://pypi.org/project/addict/
+[google-cloud-pubsub]: https://pypi.org/project/google-cloud-pubsub/
+[requirements.txt]: https://github.com/SchedMD/slurm-gcp/blob/v5.0.3/scripts/requirements.txt
+
+### Manual Configuration
+This module *does not* complete the installation of hybrid partitions on your
+slurm cluster. After deploying, you must follow the steps listed out in the
+[hybrid.md] documentation under [manual steps].
+
+[hybrid.md]: https://github.com/SchedMD/slurm-gcp/blob/v5.0.3/docs/hybrid.md
+[manual steps]: https://github.com/SchedMD/slurm-gcp/blob/v5.0.3/docs/hybrid.md#manual-configurations
+
+### Example Usage
+The hybrid module can be added to a blueprint as follows:
+
+```yaml
+- id: slurm-controller
+  source: ./community/modules/scheduler/schedmd-slurm-gcp-v5-hybrid
+  kind: terraform
+  use:
+  - debug-partition
+  - compute-partition
+  - pre-existing-storage
+  settings:
+    output_dir: ./hybrid
+    slurm_bin_dir: /usr/local/bin
+    slurm_control_host: static-controller
+```
+
+This defines a HPC module that create a hybrid configuration with the following
+attributes:
+
+* 2 partitions defined in previous modules with the IDs of `debug-partition` and
+  `compute-partition`. These are the same partition modules used by
+  [schedmd-slurm-gcp-v5-controller].
+* Network storage to be mounted on the compute nodes when created, defined in
+  `pre-existing-storage`.
+* `output_directory` set to `./hybrid`. This is where the hybrid
+  configurations will be created.
+* `slurm_bin_dir` located at `/usr/local/bin`. Set this to whereever the slurm
+  executables are installed on your system.
+* `slurm_control_host`: The name of the on premise host is provided to the
+  module for configuring NFS mounts and communicating with the controller after
+  VM creation.
+
+[schedmd-slurm-gcp-v5-controller]: ../schedmd-slurm-gcp-v5-controller/
+
+### Assumptions and Limitations
+**Shared directories from the controller:** By default, the following
+directories are NFS mounted from the on premise controller to the created cloud
+VMs:
+* /home
+* /opt/apps
+* /etc/munge
+* /usr/local/slurm/etc
+
+The expectation is that these directories exist on the controller and that all
+files required by slurmd to be in sync with the controller are in those
+directories.
+
+If this does not match your slurm cluster, these directories can be overwritten
+with a custom NFS mount using [pre-existing-network-storage] or by setting the
+`network_storage` variable directly in the hybrid module. Any value in
+`network_storage`, added directly or with `use`, will override the default
+directories above.
```

Comment:
nit: this last line maybe should be in bold.

Reply:
Done